### PR TITLE
Update documentation for Secrets

### PR DESCRIPTION
**Pull request by bdougie**
I know this project is still in active development but I noticed some blind spots in the documentation. I am starting to leverage this to provide some GraphQL examples for the Netlify team to play with and want to add some clarity to this. 

For more context: I created a forum questions -- which I answered as well. https://forums.meteor.com/t/accessing-headers-secrets-in-the-apollo-launchpad/37743

### CODE REVIEW

**Comment by stubailo** [comment]
Thanks a lot! This looks great.

File: docs.md [hunk 1]
```diff
@@ -10,6 +10,28 @@ Every pad on Launchpad is owned by the user that created it. The code and endpoi
...
 
+Adding a secret can be done by clicking the "secrets" button in the lower right of the launchpad dashboard and enter a key and value for your secret. 
+![launchpad access key modal](http://i.imgur.com/kKUUSuy.png)
+
+Accessing your secrets within your code can be done by exposing the secrets or headers object in context. See the [CurrentWeather](https://launchpad.graphql.com/5rrx10z19) example for a live example.
+```js
+const resolvers = {
+  Query: {
+    user: (root, args, context) => {
+      return fetch(`https://api.netlify.com/api/v1/user?access_token=${context.secrets.ACCESS_TOKEN}`)
```
Comment (stubailo) [inline docs.md]: Do you have a Netlify launchpad we could link to?

Comment (bdougie) [inline docs.md]: Yeah, still working on improving the Schema but it is here: https://launchpad.graphql.com/m1wl39p39

**Comment by stubailo** [comment]
Thanks @bdougie! Great improvement :]

**Pull request merged**